samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/ItemContentsBean.java

```diff
@@ -1390,6 +1390,24 @@ public String getAnswerKeyTF() {
 	return answerKey;
   }
 
+  public String getAnswerKeyCalcQuestion() {
+	String answerKey = "";
+	if(itemData!=null){
+		answerKey = itemData.getAnswerKey();
+		String keys[] = answerKey.split(",");
+		answerKey = "";
```

Comment:
This overwrites the value assigned two lines earlier ( line 1396, answerKey = itemData.getAnswerKey()). Is that intended? If so, the line 1396 is unnecessary.

Also, line 1394 sets answerKey = "" as well.